### ISSUE TITLE
Getting started guide is missing

### ISSUE THREAD

**Issue by krzkaczor** [all-contributors/all-contributors#273]
**Is your feature request related to a problem? Please describe.**
Recently I learned about all-contributors projects, I immediately looked into documentation and... got confused. There is a bot, CLI but how do I start?

**Describe the solution you'd like**
Docs should have `getting started` guide. Currently, it's only part of the landing page which is just harder to find. Also, `getting started` section in docs should be slightly more detailed.

**Comment by Berkmann18** [comment]
@krzkaczor Did you went through the sections on the left on https://allcontributors.org/docs/en/overview and saw the links in the footer of each page you went through?

**Comment by krzkaczor** [comment]
Yes... clearly marked `getting started` (step by step guide) section IS missing.

Anyway, it was just my subjective feeling that the best `getting started` guide was `How it Works` part of the landing page which is somehow weird. If you think otherwise feel free to close the issue 👍

**Comment by Berkmann18** [comment]
Fair enough, feel free to submit a PR for that.
